modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_fields.rb

```diff
@@ -160,6 +160,26 @@ def issue_decision_date_fields(index)
             day: "form1[0].#subform[#{subform}].DOBday[#{offset}]",
             year: "form1[0].#subform[#{subform}].DOByear[#{offset}]" }
         end
+
+        def boxes
```

Comment:
My gut is that since none of this data is coming from the HLR itself, this actually belongs in a private method on Structure. What do you think?

Reply:
I'm not sure I follow. None of the FormField data comes from the HLR - it's not even passed into the initializer. 

My thought for placing the boxes here was that since FormFields indicates which fields exist on the pdf, this should include "manual" fields that require bounding boxes. I didn't think it was limited to what `Structure#fill_fields` could accomplish.

Reply:
Oh, sorry! I misread the filename.